script.js

```diff
@@ -1,4 +1,154 @@
+      "<br>" +
+      " so far you draw " +
+      user_draw +
+      " times";
+  }
+  if (input == "scissors" && randomDiceNumber == 2) {
+    user_draw = user_draw + 1;
+    myOutputValue =
+      "The computer also chose scissors <br> it's a draw <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far you draw " +
+      user_draw +
+      " times";
+  }
+  if (input == "paper" && randomDiceNumber == 3) {
+    user_draw = user_draw + 1;
+    myOutputValue =
+      "The computer also chose paper <br> it's a draw <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far you draw " +
+      user_draw +
+      " times";
+  }
+
+  // comparing input and random.
+  if (input == "stone" && randomDiceNumber == scissors) {
+    user_win = user_win + 1;
+    myOutputValue =
+      "The computer chose scissors <br> you win <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far " +
+      user_name +
+      " " +
+      user_win +
+      " Computer " +
+      user_lose;
+  }
+
+  if (input == "stone" && randomDiceNumber == paper) {
+    user_lose = user_lose + 1;
+    yOutputValue =
+      "The computer chose paper <br>you lose <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far " +
+      user_name +
+      " " +
+      user_win +
+      " Computer " +
+      user_lose;
+  }
+
+  if (input == "paper" && randomDiceNumber == stone) {
+    user_win = user_win + 1;
+    myOutputValue =
+      "The computer chose stone <br> you win <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far " +
+      user_name +
+      " " +
+      user_win +
+      " Computer " +
+      user_lose;
+  }
+
+  if (input == "paper" && randomDiceNumber == scissors) {
+    user_lose = user_lose + 1;
+    myOutputValue =
+      "The computer chose scissors <br>you lose <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far " +
+      user_name +
+      " " +
+      user_win +
+      " Computer " +
+      user_lose;
+  }
+
+  if (input == "scissors" && randomDiceNumber == paper) {
+    user_win = user_win + 1;
+    myOutputValue =
+      "The computer chose paper <br>you win <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far " +
+      user_name +
+      " " +
+      user_win +
+      " Computer " +
+      user_lose;
+  }
+
+  if (input == "scissors" && randomDiceNumber == stone) {
+    user_lose = user_lose + 1;
+    myOutputValue =
+      "The computer chose stone <br>you lose <br> Now you can type scissors paper or stone to play another round!" +
+      "<br>" +
+      " so far " +
+      user_name +
+      " " +
+      user_win +
```

Comment:
Likewise, we can cut down these lines of code by combining if statements that result in almost the exact same outcome (win/lose/draw). For example:

if (input == "scissors" && randomDiceNumber == paper) || (input == "stone" && randomDiceNumber == scissors) || (input == "paper" && randomDiceNumber == stone) {
user_win = user_win + 1
...
}

we can cut down multiple lines of code and combine all into a single if statement. We can also make sure the output changes, for example:

myOutputValue = "The computer chose " + randomDiceNumber + " You chose" + input + "..."